docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.4'
+
+services:
+  db:
+    image: postgres:11-alpine
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=password
+    volumes:
+      - db-data:/var/lib/postgresql/data
```

Comment:
I think, this is redundant. Postgres image has `VOLUME` instruction within Dockerfile (https://github.com/docker-library/postgres/blob/85aadc08c347cd20f199902c4b8b4f736341c3b8/11/alpine/Dockerfile#L146).

Reply:
With `db-data:/var/lib/postgresql/data` we give a shared name for this volume, so that we could  that volume for other services, backups for instance:

```
version: "3.7"

services:
  db:
    image: db
    volumes:
      - data-volume:/var/lib/db
  backup:
    image: backup-service
    volumes:
      - data-volume:/var/lib/backup/data

volumes:
  data-volume:
```